workers/artifact_creator/main.py

```diff
@@ -0,0 +1,164 @@
+import logging
+import traceback
+import os
+import shutil
+import sys
+import config
+import tempfile
+import subprocess
+import boto3
+import gzip
+import json
+from conductor.ConductorWorker import ConductorWorker
+
+logging.basicConfig()
+log = logging.getLogger('conductor-workers.create_tenant_artifacts_images')
+log.setLevel(logging.INFO)
+
+conf = config.get_config()
+
+BUCKET_NAME = os.getenv("BUCKET")
```

Comment:
env is good, but mind you - configarparse (hidden by config.py) supports both cmd line args and envs. so each arg has an env var counterpart. so - use that instead?

Reply:
also these checks could then be easily part of check_config()